sdk/identity/azure-identity/inc/azure/identity/client_certificate_credential.hpp

```diff
@@ -14,6 +14,7 @@
 #include <azure/core/credentials/credentials.hpp>
 #include <azure/core/credentials/token_credential_options.hpp>
 #include <azure/core/internal/unique_handle.hpp>
+#include <azure/core/platform.hpp>
```

Comment:
What's requiring this include?

Reply:
It was needed for an earlier iteration. Moved to the `cpp` file.